### DISCUSSION TITLE
Standard formatted json and yaml files

### DISCUSSION THREAD

**Discussion by mdicss** [Dynatrace/dynatrace-configuration-as-code#279] [Archive] [answered]
Hi
When using download in the new cli, the downloaded files should be consistently formatted. This allows to compare two downloaded configurations. In my downloads I have e.g. for the auto-tag api:
``
    "PROCESS_GROUP_TO_SERVICE",
    "PROCESS_GROUP_TO_HOST"
``
And in the next download I have
``
    "PROCESS_GROUP_TO_HOST",
    "PROCESS_GROUP_TO_SERVICE"
``
as the only difference in a json file.

**Comment by tatomir146** [comment, marked as answer]
Hi @mdicss, download functionality is still in experimental stage and this is one of improvements marked in #179.

**Reply by mdicss** [reply]
Ok, good.
In https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/issues/179, I see only remarks about yaml formatting. I hope this will be integrated für json too.
